python/ql/src/experimental/Security/CWE-770/good.py

```diff
@@ -0,0 +1,16 @@
+from flask import Flask, jsonify, request
+import unicodedata
+
+app = Flask(__name__)
+
+
+@app.route("/good_1")
+def good_1():
+    r = request.args.get("r", "")
```

Comment:
I assume you only meant to change the variable name:
```suggestion
    r = request.args.get("file_path", "")
```

Reply:
done via https://github.com/github/codeql/pull/15319/commits/05831fea2c654fd4c454e541ed59222fca6d392e